components/illustratedmessage/stories/template.js

```diff
@@ -10,32 +11,59 @@ export const Template = ({
 	description,
 	customClasses = [],
 	useAccentColor = false,
+	hasButtons = false,
+	isHorizontal = false,
+	orientation,
+	size = "m",
 }) => html`
 	<div
 		class=${classMap({
 			[rootClass]: true,
+			[`${rootClass}--${orientation}`]: typeof orientation !== "undefined",
+			[`${rootClass}--horizontal`]: isHorizontal,
+			[`${rootClass}--size${size?.toUpperCase()}`]: typeof size !== "undefined" && size !== "m",
 			...customClasses.reduce((a, c) => ({ ...a, [c]: true }), {}),
 		})}
 	>
 		${illustrationSvgMarkup(useAccentColor)}
-		${when(
-			heading,
-			() =>
-				html`<h2
-					class="spectrum-Heading spectrum-Heading--sizeM spectrum-Heading--regular ${rootClass}-heading"
-				>
-					${heading}
-				</h2>`
-		)}
-		${when(
-			description,
-			() =>
-				html`<p
-					class="spectrum-Body spectrum-Body--sizeS ${rootClass}-description"
-				>
-					${description.map((c) => (typeof c === "function" ? c({}) : c))}
-				</p>`
-		)}
+		<div class="${rootClass}--content">
+			${when(
+				heading,
+				() =>
+					html`<h2
+						class="spectrum-Heading spectrum-Heading--regular ${rootClass}-heading"
```

Comment:
@rise-erpelding, 

Oh wow `--spectrum-illustrated-message-cjk-title-font-size` didn't appear as an unused property until I reset my VSCode 😅 It was rendering normally for me on my end. I ended up adding the language selectors within each t-shirt size. For some reason moving that line of code more up top still comes up as unused property. Switching the languages look normal now.

For the `.spectrum-Heading` and `.spectrum-Body` the only reason why I kept it there was because `main` branch had it. Removing it now doesn't break anything so it's good now.